external_file_location/tasks/sftp.py

```diff
@@ -6,12 +6,12 @@
 _logger = logging.getLogger(__name__)
 
 try:
-    from fs import sftpfs
+    from fs import osfs  # sftpfs not available in python 3
 except ImportError:
     _logger.debug('Cannot `import fs`.')
 
 
-class SftpTask(sftpfs.SFTPFS):
+class SftpTask(osfs.OSFS):
```

Comment:
Does this actually work now using SFTP? I think not. It isn't up to us to fix it, since we don't need it, but it's good to add a `TODO:` here, and a 'known issue' in the README file, as well as to remove the "SFTP" option from the selection list.